pyk4a/pyk4a.cpp

```diff
@@ -16,15 +16,12 @@ extern "C" {
                                     {1920, 1080}, {2560, 1440},
                                     {2048, 1536}, {3840, 2160},
                                     {4096, 3072}};
-    typedef struct device_container {
-        k4a_transformation_t transformation_handle;
-        k4a_calibration_t calibration_handle;
-        k4a_device_t device;
-    } device_container;
-    #define MAX_DEVICES 32
-    device_container devices[MAX_DEVICES];
 
     const char* capsule_playback_name = "pyk4a playback handle";
+    const char* capsule_device_name = "pyk4a device handle";
+    const char* capsule_calibration_name = "pyk4a calibration handle";
+    const char* capsule_transformation_name = "pyk4a transformation handle";
+    const char* capsule_capture_name = "pyk4a capture handle";
```

Comment:
Different capsule types.
Maybe we need to use CAPITAL_NAMES here?

Reply:
For variable names I agree.

Reply:
Renamed